chronicle-bom/pom.xml

```diff
@@ -46,7 +46,7 @@
         <chronicle.efx.version>2.25ea1-SNAPSHOT</chronicle.efx.version>
         <chronicle.matching.engine.version>2.25ea0</chronicle.matching.engine.version>
         <chronicle.platform.version>2.25ea0</chronicle.platform.version>
-        <chronicle.market.data.distributor.version>3.25ea0</chronicle.market.data.distributor.version>
+        <chronicle.market.data.distributor.version>3.25ea1</chronicle.market.data.distributor.version>
```

Comment:
3.25ea1-SNAPSHOT

Reply:
In fact it needs to be bumped up higher as well. For some reason there is a mismatch. MDD is on 3.25-ea1-snapshot already https://github.com/ChronicleEnterprise/Chronicle-Market-Data-Distributor/blob/ea/pom.xml